scripts/vmbuild.rb

```diff
@@ -107,10 +107,12 @@ def verify_run(output)
     imgfac_target = target.imagefactory_type
     ova_format    = target.ova_format
     compression   = target.compression_type
+    image_size    = target.image_size
     $log.info "Building for #{target}:"
 
-    tdl_name = target.name == "azure" ? "base_azure.tdl" : "base.tdl"
-    tdl_file = CFG_DIR.join(tdl_name)
+    tdl_file = CFG_DIR.join("generated", "base_#{target}.tdl")
+    File.write(tdl_file, ERB.new(File.read(CFG_DIR.join("base.tdl.erb")), 0, '-').result(binding))
```

Comment:
What are the extra parameters to `ERB.new` here for? We don't pass those for the kickstart generation.

Reply:
That adds trim mode - this allows removing extra blank lines that get added by using conditions (like <% if xxx %>).  I mentioned in https://github.com/ManageIQ/manageiq-appliance-build/issues/152 for kickstart file, but I never made the change there... 

In case we add conditions later (like using different ISOs based on a flag), I'm just adding with the trim mode now.

Without the trim mode:
```
<% if true %>
line
<% end %>
```
becomes
```

line

```

By using the trim mode with '-':

```
<% if true -%>
line
<% end -%>
```
becomes
```
line
```